.github/ISSUE_TEMPLATE/launch.md

```diff
@@ -0,0 +1,154 @@
+      if a CDN or reverse-proxy is being used.
+
+## Hosting
+- [ ] Determine who will provide the SSL certificate.
+- [ ] Setup HTTPS and test the SSL certificate.
+- [ ] Setup SSL certificate reminders.
+- [ ] Verify that the web server instance size is large enough.
+- [ ] Verify that HTTP/2 is enabled.
+- [ ] Configure redirects from `www` to the base URL (or vice-versa).
+- [ ] Create and test redirect rules.
+- [ ] Create and test reverse-proxy rules.
+- [ ] Verify that asset compression is enabled.
+
+## Infrastructure
+- [ ] Setup database backups in consultation with the devops team.
+- [ ] Create Jenkins production cache clear job if applicable.
+- [ ] Verify that Drupal's `settings.php` and `services.yml` files have the
+      correct read-only permissions.
+- [ ] Confirm syslog visibility and connectivity with external tools.
+- [ ] Verify that errors are configured to be suppressed.
+- [ ] Setup Jenkins jobs for QA (and check that PROD jenkins jobs all running)
+- [ ] Verify that Ansible playbooks follow the latest best practices.
+- [ ] Verify file uploads work.
+- [ ] Verify image styles work.
+- [ ] Run load tests to validate infrastructure.
+- [ ] Consult with the hosting provider for any platform specific launch steps
+      or advice.
+
+## Launch Prep
+- [ ] Create a roll-back plan.
+- [ ] Notify the devops team and confirm they understands what they need to do.
+- [ ] Create ordered checklist and timeline.
+- [ ] Determine who will make the go/no-go call for launch.
+- [ ] Determine who will determine if a roll-back is needed.
+- [ ] Determine who manages DNS and will update the entries for launch.
+- [ ] Lower DNS TTL to ensure quick propagation on launch day.
+- [ ] Verify that `robots.txt` is configured to allow traffic to the production
+      site (i.e. remove any changes that were made for development).
+- [ ] Verify that users of the site are aware of the cutover time/plan.
+- [ ] Create a calendar invite with a video call link that can be shared with
+      everyone who is involved with the launch.
+- [ ] Verify that user accounts have been migrated or recreated, and that users
+      have access they need.
+
+## Site Launch
+- [ ] Disable HTTP Authorization if applicable.
+- [ ] Update DNS entries.
+- [ ] Test the site for egregious issues.
+
+## Post Launch
+- [ ] Monitor site log for errors.
+- [ ] Verify caching is working as intended.
+- [ ] Verify analytics is reporting as expected.
+- [ ] Run performance tests to confirm the site works as expected.
+- [ ] Update QA environments to pull databases from the new production site.
+- [ ] Update Stage File Proxy origin URL configuration to point to the
+      production domain (e.g. `$config['stage_file_proxy.settings']['origin']`).
+- [ ] Update automated or manual tests to point to the new production domain.
+- [ ] Confirm that third party integrations are working as expected.
+- [ ] Disable migration related modules and database connectivity as applicable.
```

Comment:
I take it this refers to source database connectivity? If so, I suggest explicitly calling it that:
```suggestion
- [ ] Disable migration related modules and source database connectivity as applicable.
```